### PR TITLE
removed all 'Totals' and kept only ontario

### PR DESCRIPTION
**Pull request by benmbrew**
Trello card: https://trello.com/c/Mg3TMko9/233-make-things-smaller

I left a not on there to explain what I did

### CODE REVIEW

File: global.R [hunk 1]
```diff
@@ -124,59 +124,93 @@ get_data <- function(data_type = 'census') {
...
                                    gsub(')', '', x[2], fixed = TRUE)}))
-      # (those wih NA for the geo_code are all ontario)
+
+      # (those wih NA for the geo_code are all ontario) - give it 3500 so we can subset
+      # entirely by 4 digit geo_code
       temp_data$geo_code <- geo_codes
-      temp_data$geo_code[is.na(temp_data$geo_code)] <- 'Ontario'
+      temp_data$geo_code[is.na(temp_data$geo_code)] <- '3500'
+
+      # keep only rows that have 4 number
```
Comment (joebrew) [inline global.R:144]: @benmbrew What are the rows that don't have 4 numbers? I recall many having more than 4 - what does that mean?

**Comment by joebrew** [comment]
@benmbrew Down to merge!

**Comment by benmbrew** [comment]
Those are census tracks outside of Ontario, which we don't care about

**Comment by joebrew** [comment]
Awesome.

**Pull request merged**